src/github.com/matrix-org/go-neb/services/github/github.go

```diff
@@ -77,14 +78,69 @@ func (s *Service) requireGithubClientFor(userID string) (cli *gogithub.Client, r
 	return
 }
 
+const numberGithubSearchSummaries = 3
+const cmdGithubSearchUsage = `!github create owner/repo "search query"`
+
+func (s *Service) cmdGithubSearch(roomID, userID string, args []string) (interface{}, error) {
+	cli := s.githubClientFor(userID, true)
+	if len(args) < 2 {
+		return &gomatrix.TextMessage{"m.notice", "Usage: " + cmdGithubSearchUsage}, nil
+	}
+
+	query := fmt.Sprintf("repo:%s %s", args[0], strings.Join(args[1:], " "))
+	searchResult, res, err := cli.Search.Issues(query, nil)
+
+	if err != nil {
+		log.WithField("err", err).Print("Failed to search")
+		if res == nil {
+			return nil, fmt.Errorf("Failed to search. Failed to connect to Github")
+		}
+		return nil, fmt.Errorf("Failed to search. HTTP %d", res.StatusCode)
+	}
+
+	if searchResult.Total == nil || *searchResult.Total == 0 {
+		return &gomatrix.TextMessage{"m.notice", "No results found for your search query!"}, nil
+	}
+
+	numResults := *searchResult.Total
+	var htmlBuffer bytes.Buffer
+	var plainBuffer bytes.Buffer
```

Comment:
So far away from the first usage on `:116` :(

Reply:
done